### ISSUE TITLE
How the Softmax Activation Hinders the Detection of Adversarial and Out-of-Distribution Examples in Neural Networks

### ISSUE THREAD

**Issue by jo-kwsm** [jo-kwsm/paper_summary#31]
# INFO
## author
Jonathan Aigrain, Marcin Detyniecki

## conference or year
2020

## link
[openreview](https://openreview.net/forum?id=Byx55pVKDB)
[実装]

# 概要
敵対的サンプルやOOD データなどNN の予測を困難にする問題がある。本論文ではこれらの原因の一端はsoftmax を使用していることにあることを示し、代わりにlogit を用いることで検出精度が向上することを検証した。

# 提案手法
データセットによってlogit の分布が事なる。これはlogit 自体がデータの特徴を捉えている可能性を示唆している。
![Screenshot from 2021-02-02 21-29-22](https://user-images.githubusercontent.com/54238261/106600423-c1c33a00-659d-11eb-8576-1aef1f1fcc97.png)

logit とsoftmax の分布の違い。softmax では値域を0-1 に押し込めてしまうため情報が失われている可能性がある。
![Screenshot from 2021-02-02 19-57-49](https://user-images.githubusercontent.com/54238261/106591457-a3efd800-6591-11eb-9fe8-43639966ab56.png)

# 検証
softmax よりもlogit の方が全体的に精度が高い。
![Screenshot from 2021-02-02 19-59-05](https://user-images.githubusercontent.com/54238261/106591090-2f1c9e00-6591-11eb-8e4d-726ffead447c.png)

# 新規性
 - 異常データに対するsoftmax の影響を検証。

# 議論，展望

# Comment

# date
Feb 2, 2021
